packages/trpc/server/middlewares/sessionMiddleware.ts

```diff
@@ -75,6 +76,12 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
         select: {
           slug: true,
           metadata: true,
+          members: {
+            where: {
```

Comment:
To prevent querying all data. We only need to know if there are results or not.

```suggestion
						select: { userId: true },
            where: {
```